faqrenderer/faq.md

```diff
@@ -0,0 +1,121 @@
+
+## What is a Jupyter kernel?
+
+Jupyter kernels are the backend code which allow you to execute the code you write within Jupyter software like notebooks and consoles. For instance, if you have ever used JupyterLab locally for python programming then you have made use of the [ipython kernel](https://ipython.org/). Kernels are what change your notebook from a static text page to interactable programming shells. This is why you must include a kernel in your conda environment to run the environment as a notebook.
+
+Kernels have a runtime which is independent of any given notebook, console, or terminal, and you can find more about [managing kernels](https://jupyterlab.readthedocs.io/en/stable/user/running.html) from the official JupyterLab documentation. You generally should not have to worry about stopping and restarting kernels, but it may be useful for certain debugging purposes. A full list of Jupyter Kernels is available [here](https://github.com/jupyter/jupyter/wiki/Jupyter-kernels).
+
+## How do I restart my JupyterLab server?
+
+On Libretexts JupyterHub, you spawn servers that provide you with JupyterLab software and you execute your code using kernels. If you are actively running code and something unexpected occurs, [restarting the kernel](https://jupyterlab.readthedocs.io/en/stable/user/running.html) can often be an easy solution. However, sometimes the problem may be deeper than that, especially if some of your files were incorrectly modified. In this case, it could be a good idea to restart your server. 
+
+Go to File->Hub Control Panel and you will be brought to a page with "Stop Server" and "My Server" buttons. Press Stop Server and you should see that that the My Server button will dim; this means that the server is stopping. When the server has stopped (this should take less than 15 seconds), the webpage will prompt you with a "Start My Server" option. Press it, launch the server, and then you will be brought back to the Server Options page. Proceed with the environment of your choice, and you will now have a fresh server.
+
+By default, your server will always shutdown after 1 hour of inactivity. All files which you wish to modify and save across restarts must be located in your `/home/jovyan` directory; everything else will be wiped. Think of these user files as being stored in a cloud, and each time you start JupyterLab, we provide you with a brand new server which has downloaded your specific files. For more technical information on how your files are stored, see the Jupyterhub for Kubernetes documentation [here](https://zero-to-jupyterhub.readthedocs.io/en/latest/customizing/user-environment.html#about-user-storage-and-adding-files-to-it). 
```

Comment:
I think telling users that "everything else will be wiped" is worrisome. Just tell them that everything stored in /hom/jovyan is kept, so don't store files in other locations.

Reply:
We should tell them that files are backup weekly/daily/? by us, but they should download any important files and back them up themselves.

Reply:
We'll have to check with Richard about a backup schedule. Last time that the backup was being preformed, it basically overloaded and crashed the whole cluster. This happened last Monday if I recall correctly; we discussed it briefly during that meeting. I'll make the other changes though.

Reply:
@rfeltstykket What's the status on the hen backup? How often do you plan to run it?